include/dali/core/bitmask.h

```diff
@@ -245,6 +245,41 @@ class bitmask {
     return storage & mask;
   }
 
+  void append(const bitmask &other) {
+    if (empty()) {
+      *this = other;
+      return;
+    }
+
+    int last_bit_idx = bit_idx(size_);
```

Comment:
```suggestion
    int first_free_bit_idx = bit_idx(size_);
```
I think this value tell where to write, not where is the last valid value.

Reply:
Fixed.